review/src/androidLibMain/kotlin/ReviewManager.kt

```diff
@@ -47,7 +48,7 @@ actual class ReviewManager(
         private val type: Type = Type.Live,
         private val context: Context = ApplicationHolder.applicationContext,
         private val lifecycleManagerObserver: LifecycleManagerObserver = LifecycleManagerObserver()
-    ) : LifecycleSubscribable by lifecycleManagerObserver {
+    ) : LifecycleSubscribableMarker, LifecycleSubscribable by lifecycleManagerObserver {
```

Comment:
`LifecycleSubscribable` extends `LifecycleSubscribableMarker`, so that shouldn't be necessary.

Reply:
Tell that to the compiler :) it refuses to let it be marked `actual` without this explicit declaration.